### DISCUSSION TITLE
Using ANCOMBC2 to test Correlations with Serum Markers

### DISCUSSION THREAD

**Discussion by JLammons14** [FrederickHuangLin/ANCOMBC#223] [General]
I had a question about the application of ANCOMBC2

I am curious to know if it would be appropriate to use ANCOMBC2  to test the correlation between microbial abundance and serum marker concentration. I ask this because the serum marker I am interested in would be unlikely to effect the composition of the microbiome, so I wasn't sure if its inclusion in the analysis would through off the bias correction. 

I am also wanted to know if I could use the bias corrected log abundance from prior analysis that included biologically relevant confounders and group variables for downstream linear modeling?  

Thanks

**Comment by FrederickHuangLin** [comment]
Hello @JLammons14,

I'm not entirely sure about the structure of the serum marker data you're working with. If it's compositional in nature, you might want to consider using the `secom` function within the `ANCOMBC` package to obtain correlations. On the other hand, if the serum marker data doesn't follow a compositional structure, you may need to derive correlations by employing bias-corrected abundance for the microbiome data, along with suitable transformations for the serum marker data.

Best regards,
Frederick
